recipes/nano/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "nano" %}
+{% set version = "2.9.6" %}
+{% set major_minor = version.rpartition('.')[0] %}
+{% set sha256 = "a373507ebb4e9307a8202fbc19b5d29718025c8ec773669349211c362545d4c6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.nano-editor.org/dist/v{{ major_minor }}/nano-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - autoconf
+    - make
+    - pkg-config
```

Comment:
Technically, the 3 packages above belong in `build`.

Reply:
Really?  That is annoying. I am not sure I fully understand the distinction between build and host, then.

Reply:
In this example, they'll be windows packages and hence in `host`. There's some more detail at https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#host why python should be in host when you are building python packages.

Reply:
I see thanks!